srcpkgs/rsvd-user/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'rsvd-user'
+pkgname=rsvd-user
+version=1.0
+revision=1
+build_style="meta"
```

Comment:
meta build style should only be used for empty packages. You can omit the build style in this case.